### ISSUE TITLE
Does this app support default terminal colors?

### ISSUE THREAD

**Issue by mjsir911** [NerdyPepper/dijo#26]
Hi,

In my `.Xresources` file I have defined my background as being slightly darker than my black color:
```
*.foreground:   #c5c8c6
*.background:   #1d1f21
*.cursorColor:  #c5c8c6

! black
*.color0:       #282a2e
*.color8:       #373b41
```

When I start up the app I see that the background actually changes color and is now gray, using my black defined color rather than my background defined color.

Is there any way for me to set the terminal default as the background color?

Thanks,

**Comment by Rochala** [comment]
You can try to experiment with themes.rs file. Cursive allows you to use _TerminalDefault_ as color value.
`
pub fn pallete_gen() -> Palette {
     let mut p = Palette::default();
     p[Background] = TerminalDefault;
     p[Shadow] = TerminalDefault;
     p[View] = TerminalDefault;
     p[Primary] = TerminalDefault;
     p[Secondary] = TerminalDefault;
     p[Tertiary] = TerminalDefault;
     p[TitlePrimary] = TerminalDefault;
     p[Highlight] = TerminalDefault;
     p[HighlightInactive] = TerminalDefault;
     return p;
 }
`
Making such change will leave command field box unchanged ( still black ). To change it you have to do some changes in _command.rs_ line 41.

**Comment by Zeioth** [comment]
It's definitely a problem in your Xresources, not a bug in dijo.

**Comment by mjsir911** [comment]
I don't believe it to be a bug that I choose higher contrast colors for my terminal to help with vision & that dijo overwrites them and makes it harder to see.

Using `themes.rs` to change the colors to default did fix everything excep the command field box, thanks!

**Comment by NerdyPepper** [comment]
Thanks for the help with this one @Rochala, v0.2.3 includes this change. 

Re: command mode, I'm afraid I can't quite fix this just yet, please see #46
